app/index.js

```diff
@@ -2380,6 +2371,157 @@ const saveWindowState = (statesOnly = false) => {
     }
 }
 
+/**
+ * Returns bookmarks file path.
+ */
+const getBookmarksFilePath = () => {
+    const setFile = getSetting("bookmarksfile")
+    if (setFile === "bookmarks" || setFile.trim() === "") {
+        return joinPath(app.getPath("appData"), "bookmarks")
+    }
+    if (isAbsolutePath(setFile)) {
+        return setFile
+    }
+    return joinPath(app.getPath("appData"), setFile)
+}
+
+ipcMain.on("import-bookmarks", event => {
+    /**
+     * @typedef {import('./renderer/bookmarks').Bookmark} Bookmark
+     * @typedef {import('./renderer/bookmarks').BookmarkData} BookmarkData
+     */
+    if (!mainWindow) {
+        return
+    }
+    dialog.showOpenDialog(mainWindow, {
+        "filters": [
+            {"extensions": ["html", "htm"], "name": "HTML"}
+        ],
+        "properties": ["openFile"]
+    // @ts-ignore
+    }).then(result => {
+        if (result.canceled) {
+            return
+        }
+        const [filePath] = result.filePaths
+        const fileContent = readFile(filePath)
```

Comment:
This only parses the first file I think? Should we allow multiple in one go? If so, this entire thing should be a function, a notification per file is alright.